typescript/forms.d.ts

```diff
@@ -927,6 +928,14 @@ export type ExternalIdGeneration =
   | EndpointConfiguration
   | ExternalIdGenerationReceiptId
 
+  
+export type FormCustomPDF = {
+  /** The Id of the custom PDF. */
+  id: number
```

Comment:
```suggestion
  id: string
```

better off making this a UUID